source/Projectile.h

```diff
@@ -55,7 +57,7 @@ class Projectile : public Body {
 
 
 public:
-	Projectile(const Ship &parent, Point position, Angle angle, const Weapon *weapon);
+	Projectile(Ship &parent, Point position, Angle angle, const Weapon *weapon);
```

Comment:
Why did we have to drop the const? All I see you do is define a pointer using this reference?

Reply:
wont work with const, I have tried